### PR TITLE
Make some methods non-static + test every method

### PR DESCRIPTION
**Pull request by jacob-andersson**

### CODE REVIEW

File: Source/NSString+HYPDate.h [hunk 1]
```diff
@@ -14,9 +14,9 @@ static NSString * const HYPTimeRangeFormat = @"%@ - %@";
 + (NSString *)hyp_dateStringFromDate:(NSDate *)date;
 + (NSString *)hyp_dateStringFromDate:(NSDate *)date withFormat:(NSString *)dateStringFormat;
 
-+ (NSString *)hyp_timeStringFromDateString:(NSString *)dateString;
```
Comment (3lvis) [inline Source/NSString+HYPDate.h:17]: Hahaha, nice! I was going to make this suggestions. :+1:

Comment (3lvis) [inline Source/NSString+HYPDate.h:17]: I would suggest pretty much the same for most of the methods that take one parameter.

Comment (jacob-andersson) [inline Source/NSString+HYPDate.h:17]: Not possible if it takes a NSDate though

**Comment by 3lvis** [comment]
I really like where this is going :)

I would suggest to make the branch names using `-` instead of camelCase

Like `improve/make-api-more-convenient`

**Pull request merged**